etl/naaccr_etl_postgresql.sql

```diff
@@ -694,8 +697,14 @@ CREATE TABLE naaccr_data_points_temp
   AND naaccr_data_points_temp.variable_concept_id = cr1.concept_id_1;
 
 
+  -- Building indexes to optimize performance
 
+  CREATE INDEX idx_cr_ndpt_record_id            ON naaccr_data_points_temp  USING btree (record_id);
+  CREATE INDEX idx_cr_ndpt_naaccr_item_number   ON naaccr_data_points_temp  USING btree (naaccr_item_number);
+  CREATE INDEX idx_cr_ndpt_naaccr_item_value    ON naaccr_data_points_temp  USING btree (naaccr_item_value);
+  CREATE INDEX idx_cr_ndpt_variable_concept_id  ON naaccr_data_points_temp  USING btree (variable_concept_id);
 
+  ANALYZE naaccr_data_points_temp;
```

Comment:
The ANALYZE statement is not part of the SQL standard, so its hard to make portable across platforms.  Is this necessary?  The same also might apply to the 'USING btree' clause.

Reply:
Yes, can you please remove 'USING betree' clause?  It won't be portable.  Also, on my own dataset, removing it makes it faster.